manager/app/src/main/java/me/weishu/kernelsu/ui/screen/ExecuteModuleAction.kt

```diff
@@ -121,7 +122,9 @@ fun ExecuteModuleActionScreen(navigator: DestinationsNavigator, moduleId: String
                     icon = { Icon(Icons.Filled.Close, contentDescription = null) },
                     onClick = {
                         navigator.popBackStack()
-                    }
+                    },
+                    modifier = Modifier
+                        .navigationBarsPadding()
```

Comment:
I think change the contentWindowInsets (line 131 for now ) to `WindowInsets.safeDrawing` without `only(WindowInsetsSides.Top + WindowInsetsSides.Horizontal) ` will be better

Reply:
For the system to decide? Sure

Reply:
![Screenshot_20241230-093043](https://github.com/user-attachments/assets/44ae045e-8115-42c7-8394-1b521f179bf6)
![Screenshot_20241230-093030](https://github.com/user-attachments/assets/2db9f1f1-b651-49f9-9f4f-31634a229e38)


But the positioning for vertical is a bit off for my personal taste, are you okay with it?

Reply:
Remove `modifier = Modifier.navigationBarsPadding() ` pls
We needn't that because `Scaffold` has already added the navigation bar insets

Reply:
> Remove `modifier = Modifier.navigationBarsPadding() ` pls We needn't that because `Scaffold` has already added the navigation bar insets

It was for when `WindowInsets.safeDrawing without only(WindowInsetsSides.Top + WindowInsetsSides.Horizontal)` was there and the button would be behind the nav bar, so since it's removed I can just get rid of the `modifier = Modifier.navigationBarsPadding() `  sure